dpdd.tex

```diff
@@ -1222,20 +1226,37 @@ \subsubsection{Calibration Data}
 
-    \item A set of \emph{best seeing} coadds. One deep coadd will be created for each of the $ugrizy$ bands, using only the best seeing data (for example, using only the first quartile of the realized seeing distribution). These will be built to assist the deblending process. These coadds will be kept indefinitely and made available to the users. \dmreq{0330}\dmreq{0334}
+    The PSF model on these coadds will be propagated from the PSF models of their constituent visit-level images, and is thus expected to be discontinuous where the set of input images varies (e.g. due to CCD boundaries or rejected pixels).
+    To mitigate this effect, we are considering both building per-object postage-stamp coadds or by discretizing the inclusion or rejection of an epoch into the coadd over small predefined (and hence ensuring PSF discontinuities appear only at the well-defined boundaries between these regions).
 
-    \item A set of \emph{short-period} coadds.\dmreq{0337} These will comprise of multiple (ugrizyM) sets of yearly and multi-year coadds. Each of these sets will be created using only a subset of the data, and otherwise share the characteristics of the deep coadds described above. These are designed to enable detection of long-term variable or moving\footnote{For example, nearby high proper motion stars.} objects that would be ``washed out'' (or rejected) in full-depth coadds. \textbf{We \em do not plan to keep and make these coadds available}. We will retain and provide sufficient metadata for users to re-create them using  the Science Platform or other resources.\dmreq{0298}\dmreq{0311}
+    The six per-band coadds will be kept indefinitely and made available to the users.\dmreq{0334} \emph{Their primary purpose (after data release production is complete) is to enable the end-users to apply alternative object characterization algorithms, perform studies of diffuse structures, and for visualization}.
 
-    \item One (ugrizyM) set of PSF-matched coadds.\dmreq{0335} These will be used to measure colors and shapes of objects at ``standard'' seeing. \textbf{\emph{We do not plan to keep and make these coadds available}}. We will retain and provide sufficient metadata for users to re-create them using Science Platform  or other resources.
+    \item A set of \emph{template coadds}.  These will be used for image differencing in at least that data release, and when possible will be used in the following season's Prompt Processing.  They will be retained and made available at least during the period in which they are used by Prompt Processing.  See also \S~\ref{sec:templates}.
```

Comment:
I'm not sure I understand the "when possible" here.

Maybe instead of the "following season," say "they will be used as templates for Prompt Processing until the next Data Release."

Reply:
I am happy to assert that if everyone agrees it is unproblematic; "when possible" was just an escape clause in case we can't use DR coadds for some reason.  Maybe I should say, "at least after DR1" instead, since that's probably where an escape clause is most likely to be needed.  But I'm also a bit worried about guaranteeing that we will have sufficient depth to build coadds during DR1 for all areas of sky of interest in DR2 (etc., perhaps even with a tail extending to later DRs).

Reply:
Agree, prior to DR1 is when this is most likely to be problematic. We should assume that post DR1 we will use DR templates and deal with any issues then.  Maybe even add a statement that prior to DR1, we will not have templates covering the full sky and options for image differencing in DR1 in the absence of templates from a DR are being investigated.